oe_content.info.yml

```diff
@@ -10,6 +10,8 @@ dependencies:
   - drupal:datetime
   - rdf_skos:rdf_skos
   - maxlength:maxlength
+  - oe_content:oe_content_sub_entity
+  - entity_reference_revisions:entity_reference_revisions
```

Comment:
This now needs to become an actual dependency of composer.json and, by consequence, it needs to be removed as an optional dependency from all sub-module's README.md.